<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-image:url(:/Icons/background); 
    background-position: center;
    background-repeat:no-repeat
}
QGroupBox {
    background-color: rgba(114, 159, 207,  225);
    border-width:1.5px;
    border-color:blue;
    border-radius:15px;
}
QRadioButton{ 
    border-width:1.5px;
    border-color:blue;
    border-radius:15px;
	font: 75 14pt &quot;URW Bookman L&quot;;
} 
QRadioButton::indicator:unchecked:hover {
      image: url(:/Icons/green_light.png);
	  width:18px;
	  height:18px;
  }
QRadioButton::indicator::checked {
   image: url(:/Icons/apply.ico);
 	width:20px;
	height:20px; 
	background-color:  rgb(52, 101, 164);
	border-radius:5px;
}
QRadioButton::checked {
	font: 800 16pt &quot;URW Bookman L&quot;;
	color: rgb(252, 233, 79);
	background-color:  rgb(52, 101, 164);
	border-radius:5px;
}
 QLabel {
	border:0px solid black; 
	font-size: 20pt ; 
	font-weight:500;
	 background-color:transparent;
}

QPushButton {
	color:rgb(46, 52, 54);
	background-color: rgb(10, 191, 56);
	font: 600 20pt &quot;Ubuntu&quot;;
	padding-left:5px;
	padding-right:5px;
	padding-top:5px;
	padding-bottom:5px;
}

/*
QDialog  { 
 	background-position: center;
	background-repeat:no-repeat;  
	background-color:  rgb(13, 10, 176);
}

QGroupBox {
	background-color: rgba(14, 59, 107, 245); 
	border: 2px solid blue;
    border-radius:15px;
}
QPushButton {
	color:rgb(46, 52, 54);
	background-color: rgb(10, 191, 56);
	font: 600 20pt &quot;Ubuntu&quot;;
	padding-left:5px;
	padding-right:5px;
	padding-top:5px;
	padding-bottom:5px;
}
 
 
QLabel {
		color:white;
		font: 600 18pt &quot;Ubuntu&quot;;
		padding-left:5px; 
} 

Line {
	background-color:  rgb(85, 87, 83);
	height:1px;
}
 

/*


QDialog {
	background-image: url(:/resources/mlv3.png);
	 background-repeat:  space;
     background-position: center center;
	background-color: rgb(180, 187, 199);
padding:0px;
margin:0px;
}

QRadioButton {
    spacing: 15px;
	font: 500 24pt &quot;Times New Roman&quot;;
	margin-top:10px;
}
QRadioButton::indicator {
    width: 18px;
    height: 18px;
	margin-left:15px;
	margin-right:15px;
	color:transparent;
	background-color:transparent;
}
QRadioButton::indicator:unchecked {
	border:2px solid black;
    width: 18px;
    height: 18px;
	margin-left:15px; 
	color:transparent;
	background-color:transparent;
}
QRadioButton::indicator:unchecked:hover {
    image: url(:/resources/greenLED.png);
}
QRadioButton::indicator:checked {
    width: 25px;
    height: 25px;
	margin-left:20px;
    image: url(:/resources/star.png); 
}
 
QRadioButton:checked { 
	font: 800 24pt &quot;Times New Roman&quot;;
	color:white;
}

QRadioButton:unchecked { 
	font: 500 20pt &quot;Times New Roman&quot;;
	color:black;
}




QPushButton {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: &quot;beige&quot;;
    font: bold 22pt; 
    padding: 6px;
	color:black;
}
QPushButton:pressed {
    background-color:  rgb(50, 140, 200);
  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E5E5E5, stop: 1 rgb(149, 149, 149));
    border-style: inset;
}

QGroupBox {
	background-color: rgba(42, 156, 255, 210);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; 
}
QGroupBox::title {
	font-size:14pt;
	font-weight:700;
	color: rgb(255, 0, 0);
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0, 85, 255), stop: 1 #FFFFFF); 

}

QLabel {
	border:0px solid black; 
	font-size: 20pt ; 
	font-weight:500;
	 background-color:transparent;
}
*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="grpBox_PasswordProtection">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lblWindowHeading">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px solid black;
padding-top:10px;
padding-bottom:5px;
background-color: rgb(211, 215, 207);</string>
        </property>
        <property name="text">
         <string>Change Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPasswordHeading">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px solid black;
padding-top:10px;
padding-bottom:5px;
background-color:transparent;</string>
        </property>
        <property name="text">
         <string>Verify Old Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lblPassWordInput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 800 36pt &quot;Times New Roman&quot;;  
background-color: rgb(255, 255, 255); 
border:3px solid rgb(10, 191, 56);
border-radius:10px;
color:black;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="maxLength">
         <number>32765</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="passwordPushButtonClear">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(49, 8, 255);</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="passwordPushButtonEnter">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>60</height>
           </size>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="passwordWindowCancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left:10px;
padding-right:10px;
background-color: transparent;
color:white;
</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
